dependency/actions/compile/entrypoint/extensions/extensions-8.1.json

```diff
@@ -0,0 +1,248 @@
+      "version": "3.3.5",
+      "md5": "128ecf6c84dd71d59c12d826cc51f0c4",
+      "klass": "PeclRecipe"
+    },
+    {
+      "name": "yaml",
+      "version": "2.2.2",
+      "md5": "22678c1238f2c6848ff7d74b780a8307",
+      "klass": "PeclRecipe"
+    },
+    {
+      "name": "solr",
+      "version": "2.5.1",
+      "md5": "29fc866198d61bccdbc4c4f53fb7ef06",
+      "klass": "PeclRecipe"
+    },
+    {
+      "name": "psr",
+      "version": "1.2.0",
+      "md5": "07695da8376002babbce528205decf07",
+      "klass": "PeclRecipe"
+    }
+  ],
+  "extensions": [
+    {
+      "name": "tideways_xhprof",
+      "version": "5.0.4",
+      "md5": "68b68cd9410e62b8481445e0d89220c0",
+      "klass": "TidewaysXhprofRecipe"
+    },
+    {
+      "name": "tidy",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "FakePeclRecipe"
+    },
+    {
+      "name": "enchant",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "EnchantFakePeclRecipe"
+    },
+    {
+      "name": "pdo_firebird",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "FakePeclRecipe"
+    },
+    {
+      "name": "readline",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "FakePeclRecipe"
+    },
+    {
+      "name": "zip",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "FakePeclRecipe"
+    },
+    {
+      "name": "gd",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "Gd74FakePeclRecipe"
+    },
+    {
+      "name": "maxminddb",
+      "version": "1.11.0",
+      "md5": "5feec07108cf7898bd0591ee54b572cd",
+      "klass": "MaxMindRecipe"
+    },
+    {
+      "name": "odbc",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "OdbcRecipe"
+    },
+    {
+      "name": "pdo_odbc",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "PdoOdbcRecipe"
+    },
+    {
+      "name": "redis",
+      "version": "5.3.7",
+      "md5": "1ed6793902214cc02467666ba69dd2be",
+      "klass": "RedisPeclRecipe"
+    },
+    {
+      "name": "memcached",
+      "version": "3.2.0",
+      "md5": "acc58fea7b7f456408a25ac927846ad0",
+      "klass": "MemcachedPeclRecipe"
+    },
+    {
+      "name": "sodium",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "SodiumRecipe"
+    },
+    {
+      "name": "phpiredis",
+      "version": "1.0.1",
+      "md5": "09a9bdb347c70832d3e034655b604064",
+      "klass": "PHPIRedisRecipe"
+    },
+    {
+      "name": "oci8",
+      "version": "3.0.1",
+      "md5": "641ebceb483fa2d95e79aea99b1350de",
+      "klass": "OraclePeclRecipe"
+    },
+    {
+      "name": "pdo_oci",
+      "version": "nil",
+      "md5": "nil",
+      "klass": "OraclePdoRecipe"
+    },
+    {
+      "name": "amqp",
+      "version": "1.11.0",
+      "md5": "3a0638223e320fc21f3b079198c24f12",
+      "klass": "AmqpPeclRecipe"
+    },
+    {
+      "name": "oci8",
+      "version": "3.2.1",
+      "md5": "309190ef3ede2779a617c9375d32ea7a",
+      "klass": "OraclePeclRecipe"
```

Comment:
`oci8` is duplicated in this file. Is that intentional? I think we probably only want one.

Reply:
we're also missing `ioncube` (referencing the original PHP 8.1 extensions file - https://github.com/cloudfoundry/buildpacks-ci/blob/master/tasks/build-binary-new/php81-extensions-patch.yml)

Reply:
> Is that intentional?

@sophiewigmore I noticed this but I didn't want to remove it because I wasn't sure if this was intentional. Also, the versions of these extensions seem old.